modules/tracker/mcserver.py

```diff
@@ -85,7 +85,7 @@ def get_favicon_path(self):
 
     def get_motd_path(self):
         if self.status:
-            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address(), self.get_name)
+            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address, self.get_name()
```

Comment:
There still would be errors with this line, line is not correctly closed!
`format(self.server_address, self.get_name()`
should be changed to
`format(self.server_address, self.get_name())`